<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbulEduWidgetContribution</class>
 <widget class="QFrame" name="AbulEduWidgetContribution">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="glContribution">
   <item row="0" column="0">
    <widget class="QLabel" name="lblRole">
     <property name="text">
      <string>2.3.1 Rôle</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cbRole"/>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>366</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lblDate">
     <property name="text">
      <string>2.3.3 Date</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QDateEdit" name="deDate">
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEntite">
     <property name="text">
      <string>2.3.2 Entité</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="AbulEduVCardComboBox" name="cbEntite">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnPlusEntite">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AbulEduVCardComboBox</class>
   <extends>QComboBox</extends>
   <header>abuleduvcardcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
